### PR TITLE
Front 19 and 20

### PR DESCRIPTION
**Pull request by cecerodrgz**
This is the new branch for 19 and 20, with everything that had to be changed before.

### CODE REVIEW

File: src/components/LoginPage/Login.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    width: fit-content;
+    padding: 50px;
+    margin: 0 auto;
+    margin-top: 50px ;
+    margin-bottom: 150px;
+    button{
+        margin-top: 10px;
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
```
Comment (SimonjShurety) [inline src/components/LoginPage/Login.scss:16]: Great work centering the submit button ;)

File: src/components/LoginPage/LoginPage.js [hunk 2]
```diff
@@ -1,12 +1,37 @@
...
 
-const LoginPage = () => {
+
+export const NewLogin = () =>{
+    function handleSubmit(event) {
+        event.preventDefault()
+        const formData = new FormData(event.target)
+        const formProps = Object.fromEntries(formData)
+        console.log('You logged in!')
+        console.log("formProp", formProps)
```
Comment (SimonjShurety) [inline src/components/LoginPage/LoginPage.js:11]: Are we keeping console logs?

**Review by SimonjShurety — APPROVED** [review]
Looks good bar perhaps removing console.logs.

File: src/components/LoginPage/LoginPage.js (hunk 2 above)
Comment (cecerodrgz) [reply src/components/LoginPage/LoginPage.js:11]: I can delete this, it was only for testing!

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]
Good job Ceci! Just one more warning I found in console: `Warning: A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component.` You can see it in console when start typing data to input fields in both components. I suppose the problem could be that initial values for this inputs are not set in the context.

File: src/components/SignupPage/SingUp.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                />
+
+                <TextInput
+                    placeholder="Confirm Password"
+                    type="password"
+                    id="signUpConfirmPassword"
+                    label="Confirm Password"
+                    textarea={false}
+                />
+
```
[the remark this answers is not in the document]
Comment (cecerodrgz) [reply src/components/SignupPage/SingUp.js:61]: I just added these. Thank you for catching that!

**Review by jpremo — APPROVED** [review]
Good job on this. Looks really nice. I'm approving this, but before you merge can you change the name of the file SingUp.js to SignUp.js? Make sure to update relevant import path in App.js.

File: src/components/SignupPage/SingUp.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (Ekaterina-Bondareva) [reply src/components/SignupPage/SingUp.js:75]: Checked! Great job!

**Review by Ekaterina-Bondareva — APPROVED** [review]

**Pull request merged**